src/core/stores/VNodes/CharNode.ts

```diff
@@ -0,0 +1,10 @@
+import { VNode, VNodeType, FormatType } from '../VNode';
```

Comment:
@dmo-odoo Here's a first working draft. What do you think of the direction it's taking?